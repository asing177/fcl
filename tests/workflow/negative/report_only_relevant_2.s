transition initial -> a;
transition a -> b;
transition b -> c;
transition c -> d;
transition d -> a; // only report `d` as counreachable and specifically not `a`

@initial
go1() {
  transitionTo(@a)
}

@a
go2() {
  transitionTo(@b)
}

@b
go3() {
  transitionTo(@c)
}

@c
go4() {
  transitionTo(@d)
}

@d
go5() {
  transitionTo(@a)
}
