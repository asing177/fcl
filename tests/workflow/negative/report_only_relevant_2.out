Workflow soundness errors:
â€¢ Dead-end state: '{d}'.